core/selection.js

```diff
@@ -355,6 +335,9 @@ class Selection {
           endNode = endNode.parentNode;
         }
         const range = document.createRange();
+        if (startOffset < 0) {
+          startOffset = 0;
+        }
```

Comment:
Hm, it's just interesting for me - did u find a real scenario showing the previous code was broken?